### PR TITLE
Fix some warnings.

### PR DESCRIPTION
**Pull request by mstratford**
Just fixes various warnings/console errors.

Cherry picked from mstratford-fixes without the breaking package updates.

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+  persistStore,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER
+} from "redux-persist";
+
+const raygunMiddleware: Middleware < {}, RootState, Dispatch < any >> = (store) => (next) => (action) => {
```
Comment (markspolakovs) [inline src/store.ts:20]: Did Prettier make the angle brackets like this? Because this is WordPress code style, which I despise.

**Review by markspolakovs — APPROVED** [review]

File: src/store.ts (hunk 1 above)
Comment (markspolakovs) [reply src/store.ts:20]: The GitHub Action says no :p

**Review by markspolakovs — CHANGES_REQUESTED** [review]
Fix the code style screm and it's good to go :)

File: src/store.ts (hunk 1 above)
Comment (mstratford) [reply src/store.ts:20]: I think I need to add auto-prettier to a VS Code plugin :P

**Pull request merged**